apis/nucleus/src/components/listbox/assets/styling.js

```diff
@@ -153,7 +154,7 @@ export default function getStyles({ app, themeApi, theme, components = [], check
     ? resolveBgImage({ bgImage: themeOverrides.background.image }, app)
     : undefined;
 
-  const searchBgColor = 'rgba(255, 255, 255, 0.7)';
+  const searchBgColor = themeType === 'light' ? 'rgba(255, 255, 255, 0.7)' : 'rgba(255, 255, 255, 0.2)';
```

Comment:
consider adding searchBgColor to the themes

Reply:
yea, looked at that a bit now. Became messy. But I think using the normal bg color + an alpha might be better.

Reply:
OK